docs/page-loaders.md.js

```diff
@@ -0,0 +1,73 @@
+process.stdout.write(`---
+keywords: server-side rendering, ssr
+---
+
+# Page loaders <a href="https://github.com/observablehq/framework/pull/1523" class="observablehq-version-badge" data-version="prerelease" title="Added in #1523"></a>
+
+Page loaders are like [data loaders](./data-loaders), but for generating (or “server-side rendering”) dynamic pages. Page loaders are programs that emit [Markdown](./markdown) to standard out, and have a double extension starting with \`.md\`, such as \`.md.js\` for a JavaScript page loader or \`.md.py\` for a Python page loader.
```

Comment:
```suggestion
Page loaders are like [data loaders](./data-loaders), but for generating (or “server-side rendering”) pages dynamically. Page loaders are programs that emit [Markdown](./markdown) to standard out, and have a double extension starting with \`.md\`, such as \`.md.js\` for a JavaScript page loader or \`.md.py\` for a Python page loader.
```

(the idea is that once the page is generated, it's just the same as any other page, static or not — the viewer has no way to tell the difference)